### PR TITLE
Notebooks/code search updates

### LINKED ISSUE #5198: Notebooks GA launch: /code-search/ page updates

### PR DESCRIPTION
**Pull request by zlonko**
This closes #5198. It updates the Code Search page by adding a section for Notebooks between the "Search Engine Built for Code" and "Move Faster with Sourcegraph" sections.

- The copy is provided here: [copy doc](https://docs.google.com/document/d/1zOctM149X0OWiwcn3Udl8bIupS2ptiaBENxSJJiPL9o/edit). 
- Please note that the [GH issue](https://github.com/sourcegraph/about/issues/5198) requests that we only use the paragraph text. The section header comes from the screenshot at the bottom of the copy doc: "Document and explore code with Notebooks"
- The video is hosted on GCP at [link](https://storage.googleapis.com/sourcegraph-assets/notebooks/Notebooks_Capture_20s.mp4).
- The shadow under the video is emulated from a similar animation on the Code Insights page for consistency.

### Testing
- Navigate to `/code-search`
- Check copy for accuracy
- Ensure that the video autoplays and loops
- Assess the column order on mobile for this section (video on top, text second)

### CODE REVIEW

**Comment by zlonko** [comment]
That sounds good! I added the class to the Install component here: 1f052f6. This is how it displays:

Screen Shot 2022-04-20 at 9 12 21 AM

**Comment by katjuell** [comment]
@zlonko @elzannewentzel  this looks good to pass to stakeholders — I'll do a final plus one when we've dealt with QA requests

File: website/src/components/Install/index.tsx
Comment (bretthayes) [inline website/src/components/Install/index.tsx:53]: Just wanted to note that this component has changed a bit in my homepage PR so we'll just have to watch out for merge conflicts.

Comment (zlonko) [reply website/src/components/Install/index.tsx:53]: Thanks for the heads up, @bretthayes. This isn't due until April 27, so we have time to incorporate those changes.

Comment (bretthayes) [reply website/src/components/Install/index.tsx:53]: No worries @zlonko! Yes, already aware of the due date. Just wanted to provide some context here in case one is merged before the other since both tickets are due on the same day.

**Comment by elzannewentzel** [comment]
@zlonko PLease see stakeholder feedback in [this](https://docs.google.com/document/d/1ViVLkAD-qORMT-jaLkBxRulvwNySYerFGVydlIT8A2c/edit) document. Only action the blocking feedback :)

**Comment by zlonko** [comment]
Thanks, @elzannewentzel! The copy updates have been added here: 5b4e2ac.

**Comment by elzannewentzel** [comment]
Thanks @zlonko - I have sent it back for final review. Please remember to hold off on deploying this until 27 April

**Comment by elzannewentzel** [comment]
@zlonko Can you deploy this on Wednesday morning (27 April) before 10 am CT please?

**Comment by zlonko** [comment]
@elzannewentzel Sure thing, I will set a reminder for before that time on Wednesday morning.

Also looping in @bretthayes as he [previously flagged](https://github.com/sourcegraph/about/pull/5270#discussion_r854754318) a potential merge conflicts with PR #5276!

**Comment by zlonko** [comment]
@katjuell This is ready for your [final review](https://github.com/sourcegraph/about/pull/5270#issuecomment-1103996633). Just a reminder that we plan to deploy this morning before 11:00 EST

**Review by katjuell — APPROVED** [review]
lgtm!

**Pull request merged**
